src/main/java/com/google/api/codegen/Inflector.java

```diff
@@ -35,4 +35,14 @@ public static String singularize(String in) {
 
     return in;
   }
+
+  // TODO (garrettjones) find an existing function that does this
```

Comment:
Maybe remove my name off of this (and on the other method too).

Reply:
:)